src/dotnet/Users.Service.Migration/Migrations/20221230103324_UpgradeAvatars.cs

```diff
@@ -26,6 +27,8 @@ protected override void Up(MigrationBuilder migrationBuilder)
 
         private async Task UpAsync(MigrationBuilder migrationBuilder)
         {
+            return; // Obsolete: applied to all of our DBs
```

Comment:
not clear why it's obsolete and when it's supposed to be removed

Reply:
Dunno. Will raise a question